src/SharedFrameworkValidation/SharedFrameworkValidation.proj

```diff
@@ -56,6 +56,41 @@
              ContinueOnError="ErrorAndStop" />
   </Target>
 
+  <UsingTask TaskName="GetNetCoreAppVersionsFromFile" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
+
+  <Target Name="GetRepoVersionHashes">
+    <GetNetCoreAppVersionsFromFile 
+              PathToVersionsFile="$(PackagesDir)Microsoft.NETCore.App/$(MicrosoftNETCoreAppPackageVersion)/Microsoft.NETCore.App.versions.txt">
+      <Output TaskParameter="VersionHashes" ItemName="VersionHashes" /> 
+    </GetNetCoreAppVersionsFromFile>
+
+    <PropertyGroup>
+      <CoreFxHash Condition="'%(VersionHashes.Identity)' == 'corefx'">%(VersionHashes.VersionHash)</CoreFxHash>
+      <CoreFxCopyLocation>$(BinDir)corefxCopy</CoreFxCopyLocation>
+      <_RepoURL>https://github.com/dotnet/corefx.git</_RepoURL>
+    </PropertyGroup>
+
+    <!-- Windows script -->
+    <ItemGroup>
+      <_CloneRepoLines Include="REM @echo off" />
+      <_CloneRepoLines Include="pushd $(CoreFxCopyLocation.Replace('/', '\'))" />
+      <_CloneRepoLines Include="git clone $(_RepoURL)" />
```

Comment:
Wasn't the plan to just do a scoped checkout? I have a feeling if you do another clone under the bin folder which is under the outter git folder then somethings might be strange I don't know if git likes nested repo's (although I've not really tried).